integration-tests/hibernate-search-orm-elasticsearch/src/test/java/io/quarkus/it/hibernate/search/orm/elasticsearch/devservices/HibernateSearchElasticsearchDevServicesDisabledExplicitlyTest.java

```diff
@@ -34,7 +34,7 @@ public Map<String, String> getConfigOverrides() {
                     "quarkus.hibernate-search-orm.schema-management.strategy", "none",
                     // This version does not matter as long as it's supported by Hibernate Search:
                     // it won't be checked in this test anyway.
-                    "quarkus.hibernate-search-orm.elasticsearch.version", "7.5",
```

Comment:
I think here the idea was to use an "old" version that didn't match the default one, so maybe use 8.8 ?